.github/workflows/cd-syft-dev.yml

```diff
@@ -70,20 +70,10 @@ jobs:
         id: buildx
         uses: docker/setup-buildx-action@v3
 
-      - name: Install Azure CLI
-        run: |
-          curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
-          az version
-
-      - name: Login to Azure CLI
-        uses: azure/login@v1
-        with:
-          creds: ${{ secrets.AZURE_CREDENTIALS_GITHUB_CI }}
```

Comment:
After our testing after merge, could we also remove the secret and the Service Principal Account on Azure?